src/thing_creature.c

```diff
@@ -1111,6 +1112,8 @@ void reapply_spell_effect_to_thing(struct Thing *thing, long spell_idx, long spe
         creature_set_speed(thing, 0);
         break;
     case SplK_Heal:
+    // fall-through
```

Comment:
// fall-through is not needed here, only if after some code you want to continue.

Reply:
Done.